test/internal/utils.go

```diff
@@ -40,7 +40,14 @@ func NewTestContext(env ...string) (tc TestContext, err error) {
 
 // InstallOLM runs 'operator-sdk olm install' and returns any errors emitted by that command.
 func (tc TestContext) InstallOLM() error {
-	cmd := exec.Command(tc.BinaryName, "olm", "install", "--timeout", "4m")
+	err := tc.InstallOLMVersion("latest")
+	return err
+}
+
+// InstallOLM runs 'operator-sdk olm install' for specific version
+// and returns any errors emitted by that command.
+func (tc TestContext) InstallOLMVersion(version string) error {
```

Comment:
Have modified this, so that we can pass specific versions of olm to install. Can be used in future for testing specific versions.